TramsDataApi/Controllers/EstablishmentsController.cs

```diff
@@ -92,5 +94,25 @@ public ActionResult<List<EstablishmentSummaryResponse>> SearchEstablishments([Fr
             _logger.LogDebug(JsonSerializer.Serialize<IList<EstablishmentSummaryResponse>>(establishments));
             return Ok(establishments);
         }
+
+        [HttpGet]
```

Comment:
Might be worth adding a unit test for this new method. I think there are some examples available here - https://github.com/DFE-Digital/trams-data-api/blob/main/TramsDataApi.Test/Controllers/EstablishmentsControllerTests.cs

Reply:
Ah, I have done there are some in this PR. Thanks for taking a look btw!

Reply:
Yes, you're right. My bad.